### PR TITLE
Implemented reading for Orlov and Rolle.

### PR DESCRIPTION
**Pull request by cmoesgaard**
Generalized existing reading code slightly to facilitate this.

### CODE REVIEW

File: mora/service/employee.py [hunk 1]
```diff
@@ -22,9 +22,8 @@
 from . import common
 from .association import (ASSOCIATION_KEY, create_association,
```
Comment (danchr) [inline mora/service/employee.py]: Skal denne her også ind i `keys.py`?

File: mora/service/engagement.py [hunk 2]
```diff
@@ -216,7 +211,10 @@ def get_employee_id(effect):
         return effect['relationer']['tilknyttedebrugere'][-1]['uuid']
 
     def get_unit_id(effect):
-        return effect['relationer']['tilknyttedeenheder'][-1]['uuid']
+        try:
+            return effect['relationer']['tilknyttedeenheder'][-1]['uuid']
+        except (KeyError, IndexError):
+            return None
```
Comment (danchr) [inline mora/service/engagement.py]: Could you add a comment explaining which cases trigger this? It's not obvious to me.

File: mora/service/employee.py (hunk 1 above)
Comment (cmoesgaard) [reply mora/service/employee.py]: Den ligger i øvrigt allerede i keys, dét der er bare en reference til referencen i keys.py.
Jeg fikser det.

File: mora/service/engagement.py [hunk 3]
```diff
@@ -231,6 +229,70 @@ def get_title_id(effect):
...
+            }
+        }
+
+    converters = {
+        'engagement': convert_engagement,
+        'association': convert_association,
+        'role': convert_role,
+        'leave': convert_leave,
+    }
+
```
Comment (danchr) [inline mora/service/engagement.py]: Looks fine for now, but I believe we can make it even more general — we'll eventually want to have this fully configurable, I believe.

The validity assignment seems redundant, though; perhaps you could move it to the caller?

File: mora/service/engagement.py (hunk 2 above)
Comment (cmoesgaard) [reply mora/service/engagement.py]: This is relevant for 'Leave' as we only have relations to users, not org units.

Comment (danchr) [reply mora/service/engagement.py]: Precisely; it'd be nice to have a line noting that in the code.

File: mora/service/engagement.py (hunk 3 above)
Comment (cmoesgaard) [reply mora/service/engagement.py]: I agree - I thought about how to make this fully configurable and eventually expand on my rudimentary mappings and include them in this.
It's a somewhat non-trivial task, so I settled on this for now, just to actually have it working.

I agree on your validity point as well. I'll get it done.

**Pull request merged**